src/snowflake/snowpark/modin/plugin/extensions/dataframe_overrides.py

```diff
@@ -137,9 +137,15 @@ def decorator(base_method: Any):
 
 # Avoid overwriting builtin `map` by accident
 @register_dataframe_accessor("map")
-@dataframe_not_implemented()
-def _map(self, func, na_action: str | None = None, **kwargs) -> DataFrame:
-    pass  # pragma: no cover
+def _map(self, func: PythonFuncType, na_action: str | None = None, **kwargs):
+    # TODO: SNOW-1063346: Modin upgrade - modin.pandas.DataFrame functions
```

Comment:
this ticket SNOW-1063346 (https://snowflakecomputing.atlassian.net/browse/SNOW-1063346) is done. what's the TODO here?

Reply:
I think this line of comment is copied over and it is no longer valid since it is done

Reply:
good question. @sfc-gh-joshi might have insights into what work is left to remove these TODOs?

Reply:
@sfc-gh-dpetersohn originally left these comments in the code to identify which functions have differing implementations from the upstream modin versions, but I ended up using the ticket to track moving these methods into `dataframe_overrides.py` from the old `snowflake/snowpark/modin/pandas/dataframe.py` file. We can remove the TODO when we remove the override definition (i.e. we match the upstream modin implementation), though this might not be possible or desirable in all cases.